src/js/components/CheckboxTable.js

```diff
@@ -164,7 +166,7 @@ class CheckboxTable extends React.Component {
         buildRowOptions={this.getTableRowOptions}
         className={tableClassSet}
         columns={columns}
-        colGroup={this.getColGroup()}
+        colGroup={getColGroup()}
```

Comment:
Can we just add this to `propTypes`? And maybe add a noop as the `defaultProp` as I don't think we'll need it.